std/file.d

```diff
@@ -3246,6 +3246,41 @@ if ((isSomeFiniteCharInputRange!RO || isConvertibleToString!RO) &&
     }
 }
 
+version(Windows)
+{
+    import core.sys.windows.w32api:_WIN32_WINNT;;
```

Comment:
spaces `w32api : _WIN32_WINNT;`, also there is a duplicate `;`

Reply:
All done